lib/rubygems/package/tar_reader/entry.rb

```diff
@@ -39,6 +54,12 @@ def bytes_read
   # Closes the tar entry
 
   def close
+    return true if closed?
```

Comment:
:thinking: `close` should return `nil`, but since it already returns `true`, IMHO it is ok to keep it as is. I can open additional PR to inline this with `IO#close` later if needed.

Reply:
Makes sense. I preserved the old behavior which would have returned true. Makes more sense to behave like IO#close. I'll update it here.